gc/base/standard/ParallelGlobalGC.cpp

```diff
@@ -1092,6 +1119,9 @@ MM_ParallelGlobalGC::internalPostCollect(MM_EnvironmentBase *env, MM_MemorySubSp
 	MM_GlobalCollector::internalPostCollect(env, subSpace);
 
 	tenureMemoryPoolPostCollect(env);
+	if (env->_cycleState->_gcCode.shouldClearHeap()) {
```

Comment:
put a comment that clearing pass should be done before cycle end is reported, where heap walks/fixups are reported